tgui/packages/tgui/interfaces/common/WirePanel/index.tsx

```diff
@@ -0,0 +1,191 @@
+
+import { useBackend } from "../../../backend";
+import { Box, Button, Collapsible, Dimmer, Icon } from "../../../components";
+import type { WireData, WirePanelData } from './type';
+import { WirePanelActions, WirePanelControlLabels, WirePanelControls, WirePanelCoverStatus } from './const';
+import { BooleanLike } from "common/react";
+import { Fragment } from "inferno";
+
+export const WirePanel = (props, context) => {
+  const { act, data } = useBackend<WirePanelData>(context);
+  const { wire_panel } = data;
+
+  const act_wire = (wire_index: number, action) => {
+    act("actwire", { wire_index, action });
+  };
+
+  return (
+    <Box className="wire_panel">
+      {
+        wire_panel.wires.map((wire, i) => {
```

Comment:
Wires have a distinct order that is the same for all machines using the same panel definition within a round. Is there a good way to handle that ordering without using a numerical key?